config/default/ActivityMonitoringService.conf.php

```diff
@@ -10,4 +10,5 @@
 return new ActivityMonitoringService([
     ActivityMonitoringService::OPTION_ACTIVE_USER_THRESHOLD => 300,
     ActivityMonitoringService::OPTION_COMPLETED_ASSESSMENTS_AUTO_REFRESH => 30,
+    ActivityMonitoringService::OPTION_ASSESSMENT_ACTIVITY_AUTO_REFRESH => 60,
```

Comment:
The filter should be a MM-DD format for all but 'last day' option. I'm not currently receiving this error. I'll have to look into why this is happening for you.